build/package/Symbols.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)/Symbols.targets" />
+  
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+<!-- supress the creation of the directory structure output dirs -->
```

Comment:
nit: suppress

Reply:
Seems like it should be moved up a line, too, (it's describing `IncludeBuildOutput`, right?) and indented to fit.

Reply:
@dagood 
@mmitche
That's a 'todo' note for me; it's not going in the final merge - when the restore is executed, there are empty directories created in the bin directory - our output directory for all projects - I wanted to suppress the creation of these directories.

Reply:
Additional testing reveals that the 'dotnet restore' is not necessary to produce 'symbols.nupkg'' removing...

Reply:
The statement above is incorrect; the 'dotnet restore' IS necessary to produce 'symbols.nupkg'